src/robot/libraries/Screenshot.py

```diff
@@ -31,6 +31,7 @@
 else:
     try:
         import wx
+        wx_app = wx.App(False)  # Linux Python 2.7 must exist on global scope
```

Comment:
I'm slightly worried will this be slow or reserve unnecessary resources if wx isn't in the end used for taking screenshots. Could you test would just using `wx_app = None` here and using the code below later work as well? We'd only create the app reference when actually needed.

```python
def _wx_screenshot(self, path):
    global wx_app
    if wx_app is None:
        wx_app = wx.App(False)
    # ...
```

Reply:
It does not work. Only having the  App created in global scope, like I propose.

Reply:
What I proposed above _does_ create a global app reference.

Reply:
I know, I even tried declaring it in the import wx block, but the problem still happened.

Reply:
This sounds super strange and I'm still somewhat worried about the global `wx.App` reference. Because the app references is only created if `wx` can be imported I guess that's ok. We also use wxPython for taking the screenshot if it's available by default.